shared/schemas/workflows.yml.json

```diff
@@ -2,11 +2,27 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "$id": "https://quiltdata.com/workflows/config/1",
   "type": "object",
-  "required": [
-    "version",
-    "workflows"
-  ],
+  "required": ["version", "workflows"],
   "additionalProperties": false,
+  "definitions": {
+    "handleTemplate": {
+      "type": "string",
+      "default": "",
+      "description": "Template for pre-filling package handle, use <%= username %> or <%= directory %> for username and parent directory substitutions"
+    },
+    "packageHandle": {
+      "type": "object",
+      "description": "Map of contexts triggering different templates",
+      "properties": {
+        "files": {
+          "$ref": "#/definitions/handleTemplate"
+        },
+        "packages": {
+          "$ref": "#/definitions/handleTemplate"
+        }
+      }
+    }
+  },
   "properties": {
     "version": {
       "const": "1"
```

Comment:
@sir-sigurd I don't know should I increment version? I've added additional property `package_handle`, probably version should be `1.1` now?